sympy/core/power.py

```diff
@@ -1741,8 +1741,10 @@ def _eval_difference_delta(self, n, step):
 def _eval_is_ge(a, b):
     from sympy.core.relational import is_ge
     if is_ge(a.base, S.One) and is_ge(b.base, S.One):
-        if is_ge(a.base, b.base) and is_ge(a.exp, b.exp):
-            return True
+        base_greater = is_ge(a.base, b.base)
+        exp_greater = is_ge(a.exp, b.exp)
+        if (not base_greater is None) and (not exp_greater is None):
+            return base_greater and exp_greater
```

Comment:
I guess this could be
```
if fuzzy_and([base_greater, exp_greater]):
    return True
```
Is the logic otherwise correct when either/both of `base_greater` and `exp_greater` are False?

Reply:
actually, it needs to be just:

```python
return fuzzy_and([base_greater, exp_greater])
```

The reason? is that if fuzzy_and is false, we want it to return false, and fuzzy_and return None if one or both are none.
Yeah, this seems like the correct logic to me.

Reply:
Testing with this PR I get:
```julia
In [7]: from sympy.core.relational import is_ge                                                                                                               

In [8]: a = Pow(2, 10, evaluate=False)                                                                                                                        

In [9]: b = Pow(3, 3, evaluate=False)                                                                                                                         

In [10]: a                                                                                                                                                    
Out[10]: 
 10
2  

In [11]: b                                                                                                                                                    
Out[11]: 
 3
3 

In [12]: is_ge(a, b)                                                                                                                                          
Out[12]: False

In [13]: a > b                                                                                                                                                
Out[13]: True
```
If the base is greater and the exponent is greater then we can return true. If the base is less and the exponent is less then we can return false. If the base is greater but the exponent is less or vice-versa we need to do something else.

How is is that `a > b` evaluates to True? I would have expected that it would get its value from `is_ge`.

Reply:
> gets its value from the negation of <= which swaps the arguments of is_ge

so 
```python
is_ge(b, a)
```

evaluates to False. And the negation of False is true.

Reply:
We don't need to cover all cases. It is important though that any cases that are not covered result in `None` rather than `True/False`. In ternary fuzzy logic `None` is always a valid answer so we should always default to that.